kubejs/startup_scripts/registry/item_registry.js

```diff
@@ -458,3 +460,12 @@ StartupEvents.registry('item', event => {
             .textureJson({ layer0: `kubejs:item/circuits/universal/${universal_circuit}_universal_circuit` })
     })
 })
+
+ItemEvents.toolTierRegistry(event => {
+    event.add("dev", (tier) => {
+        tier.enchantmentValue = 42
+        tier.level = 42
+        tier.uses = 42690
+        tier.speed = 12
+    })
+})
```

Comment:
Unrelated change?

Reply:
It looks like both of these are artifacts of a bad rebase. Fixed.